docs/en/integrations/data-visualization/powerbi-and-clickhouse.md

```diff
@@ -4,107 +4,123 @@ slug: /en/integrations/powerbi
 keywords: [ clickhouse, powerbi, connect, integrate, ui ]
 description: Microsoft Power BI is an interactive data visualization software product developed by Microsoft with a primary focus on business intelligence.
 ---
-
 import ConnectionDetails from '@site/docs/en/_snippets/_gather_your_details_http.mdx';
 
 # Power BI
 
-Power BI can load the data from ClickHouse Cloud or on-premise deployment using either
-the [ODBC driver](https://github.com/ClickHouse/clickhouse-odbc)
-or [ClickHouse Native connector](https://github.com/ClickHouse/power-bi-clickhouse). Both methods support Load mode, but
-the latter also supports Direct Query mode which eliminates the necessity to load the entire table.
+Microsoft Power BI can query or load into memory data from [ClickHouse Cloud](https://clickhouse.com/cloud) or a self-managed deployment.
 
-This tutorial will guide you through the process of loading data using either of these methods.
-<br/>
-<br/>
-<br/>
+There are several flavours of Power BI that you can use to visualise your data:
+
+* Power BI Desktop: A Windows desktop application for creating Dashboards and Visualisations
+* Power BI Service: Available within Azure as a SaaS to host the Dashboards created on Power BI Desktop 
+
+Power BI requires you to create your dashboards within the Desktop version and publish them to Power BI Service. 
+
+This tutorial will guide you through the process of:
+
+* [Installing the ClickHouse ODBC Driver](#install-the-odbc-driver)
+* [Installing the ClickHouse Power BI Connector into Power BI Desktop](#install-clickhouse-connector)
+* [Querying data from ClickHouse for visualistion in Power BI Desktop](#query-and-visualise-data)
+* [Setting up an on-premise data gateway for Power BI Service](#power-bi-service)
+
+## Prerequisites
+
+### Power BI Installation
+
+This tutorial assumes you have Microsoft Power BI Desktop installed on your Windows machine. You can download and install Power BI Desktop [here](https://www.microsoft.com/en-us/download/details.aspx?id=58494)
+
+### Gather your ClickHouse connection details
+
+You'll need the following details for connecting to your ClickHouse instance:
+
+* Hostname - ClickHouse 
+* Username - User credentials
+* Password - Password of the user
+* Database - Name of the database on the instance you want to connect to 
 
-# ClickHouse Native Connector
+## Power BI Desktop 
 
-## 1. Gather your connection details
+To get started with querying data in Power BI Desktop, you'll need to complete the following steps:
 
-<ConnectionDetails />
+1. Install the ClickHouse ODBC Driver
+2. Install the ClickHouse Power BI Connector
```

Comment:
As of the desktop version released a month ago, this is no longer a requirement. Do we still want to refer to this operation as installation?

Reply:
Lets test on a fresh install.

When I tried to just install the connector, I encountered the second exception hence why I asked for the exception message to be updated.

https://github.com/ClickHouse/power-bi-clickhouse/issues/30#issue-2591351196

Reply:
When installing a new version of PowerBI from scratch - the connector is there (I tried that). When updating the PowerBI desktop app, all you need to do is to restart the application to see the connector. 

The exception you mentioned related to not having an ODBC installed (it is not related to the line "Install the ClickHouse Power BI Connector" at all)

Reply:
Testing a fresh install. Please hold.

Reply:
Will check back tomorrow, I wonder if a deployment is happening as we're not appearing in the possible connector list

<img width="1016" alt="image" src="https://github.com/user-attachments/assets/988c87b9-3abf-4a8c-8597-51c1c67a8a8c">

Reply:
I've re-read your comment properly, apologies. What would you call the operation?

Reply:
Starting from version `2.137.751.0`, no action is required. We can update the second operation header to something like 'Install the ClickHouse Power BI Connector (for versions prior to 2.137.751.0)' and add a note indicating that this step is no longer necessary for newer versions.

Reply:
I've removed the section about installing the connector manually. I would rather we ask people to update to the latest to reduce the support burden on us (potentially many combinatory versions of Power BI / ODBC / ClickHouse - Cloud/On Prem). 

I will rename the section to "Get the ClickHouse Connector" to teach people where to find it. If we find that people are asking for it, we can add it back in later as an appendix or something.